pkg/azureclients/blobclient/interface.go

```diff
@@ -20,12 +20,22 @@ import (
 	"context"
 
 	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2021-09-01/storage"
+	"sigs.k8s.io/cloud-provider-azure/pkg/retry"
+)
+
+const (
+	// APIVersion is the API version for storage.
+	APIVersion = "2021-09-01"
+	// AzureStackCloudAPIVersion is the API version for Azure Stack
+	AzureStackCloudAPIVersion = "2019-03-01"
```

Comment:
do you have the supported api version list for azure stack? Thanks.  @JesusAlvarezTorres do you have a link?

Reply:
let's use `2019-06-01` per and also add this link in the comments:
https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-profiles-azure-resource-manager-versions?view=azs-2206